### PR TITLE
Add crypto currency library

### PR DESCRIPTION
**Pull request by newbyca**

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -187,5 +187,14 @@
...
     "preview": "drwnio/storytelling.png",
     "date": "2021-01-28"
+  },
+  {
+    "name": "Crypto Currencies",
+    "description": "Top ten coin logos by market cap.",
+    "author": "newbyca",
+    "url": "https://newbyca.com",
+    "source": "newbyca/storytelling.excalidrawlib",
+    "preview": "newbyca/storytelling.png",
```
Comment (dwelle) [inline libraries.json:197]:
```suggestion
    "source": "newbyca/crypto-currencies.excalidrawlib",
    "preview": "newbyca/crypto-currencies.png",
```

**Comment by dwelle** [comment]
I think Bitcoin cash doesn't use the green logo anymore:

image

what's this one?

image

File: libraries.json [hunk 2]
```diff
@@ -187,5 +187,14 @@
     "source": "drwnio/storytelling.excalidrawlib",
     "preview": "drwnio/storytelling.png",
     "date": "2021-01-28"
+  },
+  {
+    "name": "Crypto Currencies",
+    "description": "Top ten coin logos by market cap.",
```
Comment (dwelle) [inline libraries.json:193]:
```suggestion
    "description": "Several of the most popular cryptocurrencies: Bitcoin, Ethereum, Tether, Polkadot, Cardano, Chainlink, Litecoin, Bitcoin Cash, Binance Coin...",
```

Comment (dwelle) [reply libraries.json:193]: Missing the blue thing from above

Comment (lipis) [reply libraries.json:193]: It's nice to list them all actually so people could search using find in page from the browser.

Comment (newbyca) [reply libraries.json:193]: Yes, spot on. Thank you!

Comment (newbyca) [reply libraries.json]: great suggestions, thanks https://github.com/excalidraw/excalidraw-libraries/pull/43/commits/5f73c2bba88a035565dbd5b71a1d4fb7d24975f3

**Comment by newbyca** [comment]
> I think Bitcoin cash doesn't use the green logo anymore:
> 
> image
> 
> https://excalidraw.com/#json=4881929799204864,zU88i9qmD1MMkqifOgLeog (https://excalidraw.com/#json=6402659137355776,Du9RHDB8XXGZqKoPCedSZQ)

Heh, yeah I just grabbed the color off coingecko where it's green for some reason. Updated here: https://github.com/excalidraw/excalidraw-libraries/pull/43/commits/05ebd2323eca0c71853347a83edca269c7b8908d

**Comment by dwelle** [comment]
Thanks :)

**Pull request merged**